azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/HostNameSslBinding.java

```diff
@@ -88,6 +88,13 @@
             WithSslType<ParentT> withPfxCertificateToUpload(File pfxFile, String password);
 
             /**
+             * Use an existing certificate in the resource group.
+             * @param certificateName the name of the certificate
+             * @return the next stage of the definition
+             */
+            WithSslType<ParentT> withExistingCertificate(String certificateName);
```

Comment:
can the certificate only come from the same resource group?  (is that why it's a name not resource id?)